pkg/virt-launcher/virtwrap/network/podnic.go

```diff
@@ -27,7 +27,7 @@ import (
 	"kubevirt.io/client-go/log"
 	"kubevirt.io/client-go/precond"
 	"kubevirt.io/kubevirt/pkg/network/cache"
-	dhcpconfigurator "kubevirt.io/kubevirt/pkg/network/dhcp"
+	"kubevirt.io/kubevirt/pkg/network/dhcp"
```

Comment:
I don't understand why you're replacing the pkg import. There are 2 dhcp pkgs: one with the configurator, another with the implementation of the DHCPv4 server. Without the suffix, you don't know what you're using in the middle of the code.

Furthermore, if you don't change it, there isn't a single change in this file in this commit. This is artificially bloating the changes on this commit without reason.